src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,14 @@
 import React from 'react';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+import { MovieCard } from '../MovieCard/MovieCard';
+
+import { Movies } from '../../types/Movies';
+
+export const MoviesList: React.FC<Movies> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <MovieCard {...movie} />
```

Comment:
You should add unique key for each component
```suggestion
      <MovieCard {...movie} key={...}/>
```